uk_tax_23-24.py

```diff
@@ -270,19 +285,35 @@ def plot_tax_savings_3d(salary_top_range=salary_top_range, max_voluntary_contrib
         plt.show()
 
 if __name__ == "__main__":
-    if len(sys.argv) == 2:
+    # Expecting 1 or 2 arguments - 1 for salary (e.g 120000) and 1 for Student Loan Plan Number (1 or 2 currently)
+    if len(sys.argv) == 3: # All 2 arguments passed in (salary and student loan number)
+        try:
+            gross_income = float(sys.argv[1])
+            student_loan_plan_number = int(sys.argv[2])
+            if gross_income >= 0 and student_loan_plan_number >= 1:
```

Comment:
You can check if student_loan_plan is in student_loan_thresholds dictionary keys.